### PR TITLE
Add Travis to vss-tools

### PR DESCRIPTION
**Pull request by gunnarx**
Let me know what you think...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+dist: bionic
+language: python
+python: 3.8
+
+install:
+  - pip install pyyaml anytree deprecation stringcase
```
Comment (danielwilms) [inline .travis.yml:6]: I’d propose to use the requirements.txt file for the installation of the dependencies.

Comment (gunnarx) [reply .travis.yml:6]: Yes, of course. Thanks for that feedback.  I will update it.

**Comment by gunnarx** [comment]
@danielwilms Fixed to use requirements.txt and it still builds OK.
If I get approval I might squash the commits and then merge.

**Comment by danielwilms** [comment]
ready to merge @gunnarx

**Comment by gunnarx** [comment]
Ah, I forgot to squash the two travis-related commits.  Anyway, it's merged now

**Pull request merged**